### PR TITLE
allows 2d sinker

### PR DESCRIPTION
**Pull request by tcclevenger**
I edited the nsinker.cc file to only use the first two coordinates from the center points when running in 2d. Previously not implemented.

### CODE REVIEW

**Comment by gassmoeller** [comment]
Looks good to me. Could you add a testcase for this to make sure we do not break it in the future? Just copy tests/nsinker* to tests/nsinker_2d* and modify to use the 2D case. Then update the test results as described in manual section 6.8.2.

File: benchmarks/nsinker/nsinker.cc [hunk 1]
```diff
@@ -158,33 +163,27 @@ namespace aspect
...
-      double delta = 200.0;
-      double omega = 0.1;
-      double chi = 1.0;
-
-      // Centers for the sinkers provided by Cedric Thielot (pers. comm. from Dave May)
-      std::vector<Point<dim> > centers(76);
+      delta = 200.0;
+      omega = 0.1;
 
+      centers.resize(76);
```
Comment (tjhei) [inline benchmarks/nsinker/nsinker.cc]: 75?

Comment (tcclevenger) [reply benchmarks/nsinker/nsinker.cc]: right, typo.

File: benchmarks/nsinker/nsinker_2d.prm [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# This is the "NSinker" benchmark defined in May et al. (2015,G3) in the
```
Comment (tjhei) [inline benchmarks/nsinker/nsinker_2d.prm]: change this comment to just say that this is a 2d version of the originally 3d problem

Comment (tcclevenger) [reply benchmarks/nsinker/nsinker_2d.prm]: done.

**Pull request merged**